101-deploy-dsc-pullserver-to-win-server/azuredeploy.json

```diff
@@ -0,0 +1,304 @@
+        },
+        "AdminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The admin password for the VM created."
+            }
+        },
+        "WindowsOSVersion": {
+            "type": "string",
+            "defaultValue": "2012-R2-Datacenter",
+            "allowedValues": [
+                "2008-R2-SP1",
+                "2012-Datacenter",
+                "2012-R2-Datacenter",
+                "Windows-Server-Technical-Preview"
+            ],
+            "metadata": {
+                "description": "The version of the Windows OS for the VM."
+            }
+        },
+        "ImagePublisher": {
+            "type": "string",
+            "defaultValue": "MicrosoftWindowsServer",
+            "metadata": {
+                "description": "The publisher of the vm."
+            }
+        },
+        "ImageOffer": {
+            "type": "string",
+            "defaultValue": "WindowsServer",
+            "metadata": {
+                "description": "The image used for the Windows vm."
+            }
+        },
+        "VmSize": {
+            "type": "string",
+            "defaultValue": "Standard_D2",
+            "allowedValues": [
+				"Standard_A0",
+				"Standard_A1",
+				"Standard_A2",
+				"Standard_A3",
+                "Standard_A4",
+                "Standard_D1",
+                "Standard_D2",
+                "Standard_D3",
+                "Standard_D4"
+			],
+            "metadata": {
+                "description": "The machine type for the server vm."
+            }
+        },
+        "dscPullSrvName": {
+            "type": "string",
+            "defaultValue": "pctst03pull",
+            "metadata": {
+                "description": "The name of the Windows vm."
+            }
+        },
+        "dscPullIPDnsName": {
+            "type": "string",
+            "defaultValue": "pctst03pubipull",
+            "metadata": {
+                "description": "The name of the public ip address"
+            }
+        },
+        "deployDSCPullServerConfigurationFile": {
+            "type": "string",
+            "defaultValue": "https://raw.githubusercontent.com/tcsatheesh/azure-quickstart-templates/master/101-deploy-dsc-pullserver-to-win-server/ConfigurePullServer.ps1.zip",
+            "metadata": {
+                "description": "The location of the pull server configuration module."
+            }
+        },
+        "deployDSCPullServerConfigurationFunction": {
+            "type": "string",
+            "defaultValue": "ConfigurePullServer.ps1\\ConfigurePullServer",
+            "metadata": {
+                "description": "The dsc configuration that implements the pull server."
+            }
+        }
+    },
+    "variables": {
+        "VirtualNetworkPrefix": "10.0.0.0/16",
+        "VirtualNetworkSubnet1Name": "Subnet-1",
+        "VirtualNetworkSubnet1Prefix": "10.0.0.0/24",
+        "VirtualNetworkSubnet2Name": "Subnet-2",
```

Comment:
This is an artifact of copy and paste from other templates :) Subnet 2 isn't actually used in the template. This and other subnet2 related variables can be removed